### DISCUSSION TITLE
🔧 Type the configuration

### DISCUSSION THREAD

**Discussion by Azema4ka** [tailwindlabs/tailwindcss#7181] [Ideas]
# Proposal
## Description
I would like to have a **check** and **autocomplete** configuration, this innovation will **improve the DX**
## Example
> tailwind.config.ts
> ```ts
>import { defineTailwindConfig } from "tailwindcss";
>
> module.exports = defineTailwindConfig({
>  content: ["./index.html", "./src/**/*.{vue,js,ts,jsx,tsx}"],
>  theme: {},
>  plugins: [],
> }) 
>```

**Comment by olivermrose** [comment]
There's already a types package: `npm i -d @types/tailwindcss`.
